.github/workflows/unit-test.yml

```diff
@@ -0,0 +1,52 @@
+name: UnitTests
+
+on:
+  pull_request:
+    branches: 
+     - "**"
+
+jobs:   
+  run-unit-tests:
+
+    runs-on: ubuntu-latest
+    container: compucorp/civicrm-buildkit:1.0
+    
+    env:
+      REPOSITORY_NAME: ${{ github.event.repository.name }}
+      CIVICRM_CORE: 5.24.6
+      CIVICRM_EXTENSIONS_DIR: /buildkit/civicrm-buildkit/build/drupal-clean/web/sites/all/modules/civicrm/tools/extensions
+    
+    services:
+      mysql:
+        image: mysql:5.7
+        env:
+          MYSQL_ROOT_PASSWORD: root
+        ports:
+        - 3306
+        options: --health-cmd="mysqladmin ping" --health-interval=10s --health-timeout=5s --health-retries=3
+
+    steps:
+
+      - uses: actions/checkout@v2
+
+      - name: Config mysql database
```

Comment:
@erawat can we be a bit more descriptive here and explain why this is necessary?